pkg/filesystem/path/windows_format.go

```diff
@@ -41,6 +43,30 @@ func (p windowsParser) ParseScope(scopeWalker ScopeWalker) (next ComponentWalker
 			}
 			return next, windowsRelativeParser{stripWindowsSeparators(p.path[2:])}, nil
 		}
+
+		if (p.path[0] == '\\' && p.path[1] == '\\') || (p.path[0] == '/' && p.path[1] == '/') {
```

Comment:
Just to double check: `\/foo\bar` is not permitted, right? Otherwise we should write something like:
```go
if (p.path[0] == '\\' || p.path[0] == '/') && (p.path[1] == '\\' || p.path[1] == '/') {
```